src/components/NavBar/Navbar.tsx

```diff
@@ -0,0 +1,28 @@
+import Web3Status from 'components/Web3Status'
+import { useWindowSize } from 'hooks/useWindowSize'
+
+import { breakpoints } from '../../nft/css/sprinkles.css'
+import * as styles from './Navbar.css'
+
+const MobileNavbar = () => {
+  return (
+    <>
+      <nav className={styles.nav} />
+      <div className={styles.mobileWalletContainer}>
```

Comment:
think you can just conditionally add this in `Navbar` without needing a separate `MobileNavbar`?

Reply:
There are a lot more differences between the two when content is added. I figured it would be best to separate now for easier addition later.

Reply:
ah in that case then i think we can leave it this way